src/main/java/com/esri/core/geometry/Geometry.java

```diff
@@ -150,6 +150,23 @@ public int value() {
 	 */
 	public abstract int getDimension();
 
+	/**
+	 * Returns an estimate of this object size in bytes.
+	 *
+	 * @return Returns an estimate of this object size in bytes.
+	 */
+	public abstract long estimateMemorySize();
+
+	protected static long estimateMemorySize(double[] attributes)
+	{
+		return attributes != null ? sizeOfDoubleArray(attributes.length) : 0;
+	}
+
+	protected static long estimateMemorySize(VertexDescription description)
```

Comment:
@mbasmanova The estimateMemorySize(VertexDescription description) method should be removed. The description instance is shared between all instances of geometry.

Reply:
@stolstov Good to know. I'll remove it.